src/Microsoft.TemplateEngine.Cli/CommandParsing/INewCommandInputExtensions.cs

```diff
@@ -41,14 +43,32 @@ internal static string ListCommandExample(this INewCommandInput command)
             return $"dotnet {command.CommandName} --list";
         }
 
-        internal static string SearchCommandExample(this INewCommandInput command, string templateName)
+        internal static string SearchCommandExample(this INewCommandInput command, string? templateName = null, IEnumerable<string>? additionalArgs = null, bool usePlaceholder = false)
         {
-            if (templateName.Any(char.IsWhiteSpace))
+            if (usePlaceholder)
             {
-                templateName = $"'{templateName}'";
+                templateName = "<TEMPLATE_NAME>";
```

Comment:
I thought about it, however since it is not localized in help and docs (as part of synopsis https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new#synopsis) suggest we not localize.